model_analyzer/result/run_config_measurement.py

```diff
@@ -380,6 +394,41 @@ def __lt__(self, other):
 
         return not self.is_better_than(other)
 
+    def is_passing_constraints(self):
+        """
+        Returns true if all model measurements pass
+        their respective constraints
+        """
+        return ConstraintManager.satisfies_constraints(
+            self._model_config_constraints, self)
+
+    def compare_measurements(self, other):
+        """
+        Compares two RunConfig measurements based on each
+        ModelConfigs weighted metric objectives and the
+        ModelConfigs weighted value within the RunConfig
+
+        Parameters
+        ----------
+            other: RunConfigMeasurement
+            
+        Returns
+        -------
+        float
+           Positive value if other is better
+           Negaive value is self is better
+           Zero if they are equal 
+        """
+        # Step 1: for each ModelConfig determine the weighted score
+        weighted_mcm_scores = self._calculate_weighted_mcm_score(other)
+
+        # Step 2: combine these using the ModelConfig weighting
+        weighted_rcm_score = self._calculate_weighted_rcm_score(
+            weighted_mcm_scores)
+
+        # Positive value means self is better than other, so invert this
```

Comment:
I think it's better to remove this comment since the positive value in this comment does not correspond to the positive value in the docstring, which can be a bit confusing.

Reply:
My goal here was to explain why I'm inverting the value before returning. It's because the inner functions return opposite polarity from what we want, I've rewritten the comment to (hopefully) make it less confusing.